portal-bridge/src/types/state/spec_id.rs

```diff
@@ -0,0 +1,61 @@
+use revm_primitives::SpecId;
+
```

Comment:
I think this is overkill

Reply:
The issue with the current approach is that if somebody swaps order of `else if` below by mistake, it will stop working.

But it's not that big deal, so up to you.

Reply:
Wouldn't that be a issue with any ordered if statement, I think I will leave it as it would be weird if someone tried to switch the order, and if it did happen it would be detected probably quickly